File: Java-OOP-Basics/Problems Course Instance 2016_2017/src/Lesson01DefiningClasses/Exercise/pr09_google/Car.java
package Lesson01DefiningClasses.Exercise.pr09_google;

class Car {
    private String model;
    private int maxSpeed;

    Car(String model, int maxSpeed) {
        this.model = model;
        this.maxSpeed = maxSpeed;
    }

    String getInfo() {
        return String.format("%s %d", this.model, this.maxSpeed);
    }
}

File: C#-OOP-Basics/Problems Course Instance 2018/DefiningClasses/Google/Car.cs
public class Car
{
    public Car() { }

    public Car(string model, int speed)
    {
        this.Model = model;
        this.MaxSpeed = speed;
    }
    public string Model { get; set; }
    public int MaxSpeed { get; set; }

    public override string ToString()
    {
        return $"{this.Model} {this.MaxSpeed}";
    }
}